### PR TITLE
Split review

### PR DESCRIPTION
**Pull request by nlisgo**
Improves: https://github.com/libero/reviewer/issues/481

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 eLife Sciences Publications, Ltd
+Copyright (c) 2019 Libero
```
Comment (diversemix) [inline LICENSE]: https://libero.pub/ - has the copyright statement to `eLife Sciences Publications, Ltd`

so which is it @BlueReZZ

**Review by diversemix — APPROVED** [review]

File: LICENSE (hunk 1 above)
Comment (BlueReZZ) [reply LICENSE]: It's eLife Sciences Publications, Ltd as Libero isn't a legal entity.

Comment (BlueReZZ) [reply LICENSE]: You can use the one in `community` as the template: https://github.com/libero/community/blob/master/LICENSE

**Review by diversemix — CHANGES_REQUESTED** [review]
Revert licence

File: LICENSE (hunk 1 above)
Comment (nlisgo) [reply LICENSE]: > It's eLife Sciences Publications, Ltd as Libero isn't a legal entity.

My bad. I had seen that some of the split repo's had `Libero` as the copyright owner and I some had `eLife Sciences Publications, Ltd`. I will address this for the reviewer repos.

**Review by diversemix — APPROVED** [review]

**Pull request merged**